domain/personallearningplan/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/personallearningplan/Goal.kt

```diff
@@ -2,9 +2,10 @@ package uk.gov.justice.digital.hmpps.domain.personallearningplan
 
 import uk.gov.justice.digital.hmpps.domain.personallearningplan.GoalStatus.ARCHIVED
 import uk.gov.justice.digital.hmpps.domain.personallearningplan.GoalStatus.COMPLETED
+import uk.gov.justice.digital.hmpps.domain.personallearningplan.dto.ReasonToArchiveGoal
 import java.time.Instant
 import java.time.LocalDate
-import java.util.UUID
+import java.util.*
```

Comment:
I don't know how your code formatter and linter has let you get away with this?
I was sure the configured lint rules objected to wildcard imports, and the code formatter was configured to not use them (or only use wildcards when there were at least 5?) or something like that?

Anyway, I guess the linter is happy 👍  ...... but its sending my OCD bananas and is eating me alive ....! 🤣

Reply:
(Don't worry; I'll try not to lose too much sleep over it! 🤣 )

Reply:
I've got everything set to a large number so it shouldn't be doing it. For some reason it's java.util so there must be a separate setting somewhere